addons/hr_expense/tests/test_expenses.py

```diff
@@ -72,3 +74,186 @@ def test_account_entry(self):
         self.assertEquals(len(self.analytic_account.line_ids), 1, "Analytic Account should have only one line")
         self.assertAlmostEquals(self.analytic_account.line_ids[0].amount, -636.36, "Amount on the only AAL is wrong")
         self.assertEquals(self.analytic_account.line_ids[0].product_id, self.product_expense, "Product of AAL should be the one from the expense")
+
+
+class TestExpenseRights(TestExpenseCommon):
```

Comment:
Well I think it is a bit too much test lines. Isn't it possible to reduce a bit the size of the code ? Maybe by re-using data and updating user groups, stuff like that.